### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by afmicc**
#### Description

* Customize welcome page

Screen Shot 2020-03-10 at 5 24 31 PM

### CODE REVIEW

File: src/pages/welcome/WelcomePage.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { Redirect, useHistory } from 'react-router-dom';
+
+import './WelcomePage.css';
+import routes from 'constants/routes';
+import smilies from 'assets/smilies.svg';
+import Button from 'components/common/button/Button';
+
+const WelcomePage = () => {
+  const authenticated = useSelector(state => state.user.auth);
+  let history = useHistory();
```
Comment (peter7z) [inline src/pages/welcome/WelcomePage.js]:
```suggestion
  const history = useHistory();
```

File: src/pages/welcome/WelcomePage.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { useSelector } from 'react-redux';
+import { Redirect, useHistory } from 'react-router-dom';
+
+import './WelcomePage.css';
+import routes from 'constants/routes';
+import smilies from 'assets/smilies.svg';
+import Button from 'components/common/button/Button';
+
+const WelcomePage = () => {
+  const authenticated = useSelector(state => state.user.auth);
```
Comment (peter7z) [inline src/pages/welcome/WelcomePage.js]: If you want...
```suggestion
  const authenticated = useSelector({ user } => user.auth);
```

File: src/pages/welcome/WelcomePage.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+.home-column-left {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, .5);
```
Comment (agch-dev) [inline src/pages/welcome/WelcomePage.css:2]: You shouldn't be using hardcoded colors, you should create a constant and use it from there

File: src/pages/welcome/WelcomePage.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+.home-list__bullet::before {
+  color: #2fbcf7;
+  content: "\2022";
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.home-list__label {
+  font-size: 14px;
```
Comment (agch-dev) [inline src/pages/welcome/WelcomePage.css:33]: should be in rem https://github.com/rootstrap/tech-guides/blob/master/css.md
check other instances of font size

File: src/pages/welcome/WelcomePage.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  line-height: 26px;
```
Comment (agch-dev) [inline src/pages/welcome/WelcomePage.css:34]: unitless, check other instances of line-height

File: src/pages/welcome/WelcomePage.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  let history = useHistory();
+  const hangleClick = () => history.push('/dashboard');
```
Comment (agch-dev) [inline src/pages/welcome/WelcomePage.js:13]: route shouldn't be hardcoded, also typo in handleClick

File: src/pages/welcome/WelcomePage.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+  if (!authenticated) {
+    return <Redirect to={routes.home} />;
+  }
+
+  return (
+    <div className="page-container">
+      <div className="column home-column-left">
+        <img src={smilies} alt="smilies"></img>
+        <span className="column__header column__header--no-transform column__header--no-bold">
```
Comment (agch-dev) [inline src/pages/welcome/WelcomePage.js:23]: Not every piece of text should be a span. Appropriate hierarchy should be set, it gives the page structure and helps a lot with accessibility. https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/HTML_text_fundamentals

File: src/pages/welcome/WelcomePage.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+          Welcome to <strong>TARGET</strong>
+        </span>
+        <span className="column__subheader">Find people near you & Connect</span>
+        <div className="home-list">
+          <div className="home-list__option">
+            <span className="home-list__bullet"></span>
+            <p className="home-list__label">
+              Create a target by clicking wherever on the map, specify the ratio and and a topic:
```
Comment (agch-dev) [inline src/pages/welcome/WelcomePage.js:31]: repeated "and"

File: src/pages/welcome/WelcomePage.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+              Travel, Dating, Music, etc.
+            </p>
+          </div>
+          <div className="home-list__option">
+            <span className="home-list__bullet"></span>
+            <p className="home-list__label">
+              <strong>TARGET</strong> will start a chat whenever you’ve a match. You can always
```
Comment (agch-dev) [inline src/pages/welcome/WelcomePage.js:38]: "you've" contraction sounds weird in this sentece.

**Review by agch-dev — COMMENTED** [review]
Awesome work! I like what you are doing with css class names!

File: src/pages/welcome/WelcomePage.js (hunk 9 above)
Comment (afmicc) [reply src/pages/welcome/WelcomePage.js:38]: I just copy and paste the text from Figma I didn't pay attention to what it says 😅

**Comment by afmicc** [comment]
I will merge this PR and create a new one with all unresolved recommendations

**Review by peter7z — APPROVED** [review]

**Pull request merged**